composer.json

```diff
@@ -13,7 +13,7 @@
         "symfony/console": "~2.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.1"
+        "phpunit/phpunit": "~4.4"
```

Comment:
I've reported #35 to replace usage of new PHPUnit method instead of increasing needed PHPUnit version.